config/recommender_config.yaml

```diff
@@ -0,0 +1,29 @@
+application: openshift-etcd
```

Comment:
Thoughts on keeping the config as part of utils to make it easy for the user to configure and run it?